### ISSUE TITLE
Double carriage return when using \r\n as line terminator

### ISSUE THREAD

**Issue by mike-at-upside** [awslabs/aws-data-wrangler#415]
**Describe the bug** 

Passing in `\r\n` as the line terminator when calling `to_csv` yields a CSV file with an extra carriage return.

**To Reproduce**

```
import awswrangler
import pandas


if __name__ == "__main__":
    bucket = "example-bucket"
    key = "example.csv"
    df = pandas.DataFrame(data={
        "reading": ["col1", "col2"],
        "timestamp": [1601379427618, 1601379427625],
        "value": [1, 2],
    })

    awswrangler.s3.to_csv(df=df,
                          path=f"s3://{bucket}/{key}",
                          index=False,
                          line_terminator="\r\n")
```

Gives the following output:

```
cat -e ~/carriage-return.csv

reading,timestamp,value^M^M$
col1,1601379427618,1^M^M$
col2,1601379427625,2^M^M$
```

Where `^M` == `\r` and `$` == `\n`. Notice the extra `^M`.

If I make `line_terminator` = `\n`, I get the following output:

```
cat -e ~/newline.csv

reading,timestamp,value$
col1,1601379427618,1$
col2,1601379427625,2$
```

Which is what I would expect.

I'm running this on MacOS but have also seen this when running in a lambda function on AWS.

Thank you!

**Comment by igorborgest** [comment]
Hi @mike-at-upside, thanks for reporting it. Awesome troubleshoot!

I've just fixed this issue in the commit above.

Do you mind to test it from our development branch and check if it helps in your use case?
The ideia is to publish this enhancement in the version `1.9.6` next weekend.

```sh
pip install git+https://github.com/awslabs/aws-data-wrangler.git@dev
```

**Comment by mike-at-upside** [comment]
That works, @igorborgest. Thanks!

**Comment by igorborgest** [comment]
Released in [1.9.6](https://github.com/awslabs/aws-data-wrangler/releases/tag/1.9.6)!
